pkg/moq/testpackages/typealiastwo/internal/typealiasinternal/typealiasinternal.go

```diff
@@ -0,0 +1,6 @@
+package typealiasinternal
+
+// we shouldn't be able to import this type directly, you need to use the alias in the parent package
+type MyInternalType struct {
```

Comment:
Does it make sense to test for a type with a type argument?

Reply:
Yes, I think that's reasonable! Updated the test to include an alias with a type argument, as well as the original one without.